pegasus/sites.v3/code.org/public/teacher-dashboard/landing.haml

```diff
@@ -4,8 +4,14 @@ theme: none
 content-type: text/ng-template
 ---
 :ruby
```

Comment:
All of this ruby code should be pulled out of the HAML and into an appropriate RB.